mission_control/navigator_missions/navigator_singleton/pose_editor.py

```diff
@@ -233,9 +228,14 @@ def as_MoveToGoal(self, linear=[0, 0, 0], angular=[0, 0, 0], **kwargs):
             **kwargs
         )
 
-    def as_MoveToGoalWaypoint(self, linear=[0, 0, 0], angular=[0, 0, 0], **kwargs):
-        return MoveToWaypointGoal(
-            target=self.as_PoseTwist(linear, angular),
+    def as_MoveGoal(self, move_type='drive', **kwargs):
+        if 'focus' in kwargs:
+            if not isinstance(kwargs['focus'], Point):
+                kwargs['focus'] = navigator_tools.numpy_to_point(kwargs['focus'])
```

Comment:
Since you're assuming its a np.array if it's not a point, maybe you should check numpy.arrayness. Ex, it would crash if someone passes a tuple.